dask/dataframe/tests/test_dataframe.py

```diff
@@ -981,6 +981,23 @@ def test_unknown_divisions():
     assert_eq(d.a + d.b + 1, full.a + full.b + 1)
 
 
+def test_sum_with_min_count():
+    dfs = [pd.DataFrame([[None, 2, 3],
+                         [None, 5, 6],
+                         [5, 4, 9]]),
+           pd.DataFrame([[2, None, None],
+                         [None, 5, 6],
+                         [5, 4, 9]])]
+    ddfs = [dd.from_pandas(df, npartitions=4) for df in dfs]
+    axes = [0, 1]
+
+    for df, ddf in zip(dfs, ddfs):
+        for axis in axes:
+            for min_count in [0, 1, 2, 3]:
+                assert_eq(df.sum(min_count=min_count, axis=axis),
+                          ddf.sum(min_count=min_count, axis=axis).compute())
```

Comment:
There's no need to call `.compute()` here.  See the bottom section of http://docs.dask.org/en/latest/develop.html#test

Reply:
I recommend that you also test `prod` here, perhaps just with a second `assert_eq` statement in the same for loop.

Reply:
Okay, thanks for the hint. I have made the tests without the call to ``.compute()`` now.